src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* `LogRecord.Attributes` and `LogRecord.StateValues` will now automatically
```

Comment:
I feel this is bit unclear from the user perspective, consider borrow some wording from https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md#140-beta4.